tidy3d/web/config.py

```diff
@@ -57,14 +54,13 @@ class WebConfig:  # pylint:disable=too-many-instance-attributes
     website_endpoint="https://tidy3d.simulation.cloud",
 )
 
-# default one to import
-DEFAULT_CONFIG = ConfigProd
+WEB_CONFIGS = {
+    "prod": ConfigProd,
+    "preprod": ConfigPreProd,
+    "uat": ConfigUat,
+    "dev": ConfigDev,
+}
 
-if os.environ.get("TIDY3D_ENV") == "dev":
-    DEFAULT_CONFIG = ConfigDev
-elif os.environ.get("TIDY3D_ENV") == "uat":
-    DEFAULT_CONFIG = ConfigUat
-elif os.environ.get("TIDY3D_ENV") == "preprod":
-    DEFAULT_CONFIG = ConfigPreProd
-elif os.environ.get("TIDY3D_ENV") == "prod":
-    DEFAULT_CONFIG = ConfigProd
```

Comment:
I don't think we want to give up on setting `TIDY3D_ENV` as an environment variable.

Reply:
Ah, right.  Ok I will add these back in, that's a good suggestion

Reply:
ok pushed a change that sets the default value of `config.web_config` to the env variable if present.